tests/testsuite/doc.rs

```diff
@@ -1168,6 +1168,39 @@ fn doc_workspace_open_help_message() {
         .run();
 }
 
+#[cargo_test]
+#[cfg(not(windows))] // `echo` may not be available
+fn doc_extern_map_local() {
+    assert!(is_nightly());
+
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+                [package]
+                name = "foo"
+                version = "0.1.0"
+            "#,
+        )
+        .file("src/lib.rs", "")
+        .build();
+
+    p.change_file(
+        ".cargo/config.toml",
+        r#"
+[doc.extern-map]
+std = "local"
+    "#,
+    );
```

Comment:
This can just be a call to `file()` above.  You might be able to fit it on one line with `.file(".cargo/config.toml", "doc.extern-map.std = 'local'")`